RELEASE.md

```diff
@@ -0,0 +1,23 @@
+# Release
```

Comment:
Please add the reference to this file into `README.md` like we have it in other libs